keyboards/sam/s80/readme.md

```diff
@@ -0,0 +1,16 @@
+# Sam S80
+
+Sam S80 TKL Keyboard
+
+![Sam S80](https://i.imgur.com/DCzBjnu.jpg)
+
+* Keyboard Maintainer: [CMM.Studio Freather](https://github.com/CMMS-Freather)
+* Hardware Supported: PCB, Atmega32u4
+
+Make example for this keyboard (after setting up your build environment):
+
+    make sam/s80:default
+
+
+For reset instruction,  use tweezer to short the top two pins to enter bootloader mode
```

Comment:
"top two pins" is a bit hard to understand. It would be best to add a more detailed description of the pins to short, including pin numbers, and the other components near those pins. Also, some readers might not understand "for reset instruction". It might be better to say "To enter the bootloader" instead...

Reply:
because it is a 1x6 shaped pins for isp flashing, the top two pins are pretty obviois.